### PR TITLE
add new commands

### PR DESCRIPTION
**Pull request by Pnom**

### CODE REVIEW

File: docs/obis-observer/commands/GetLorawanState.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+## Response
+
+### Format
+
+| Size | Type                                 | Field                              |
+| ---- | ------------------------------------ | ---------------------------------- |
+| `1`  | `byte`                               | command id = `0x25`                |
+| `1`  | [Request ID](../types.md#request-id) | request/response unique identifier |
+| `1`  | `byte`                               | downlink quality in percent        |
+| `1`  | `byte`                               | rssi of the last frame             |
```
Comment (DeadByElpy) [inline docs/obis-observer/commands/GetLorawanState.md:34]:
```suggestion
| `1`  | `byte`                               | RSSI of the last frame             |
```

File: docs/obis-observer/commands/GetLorawanState.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+| `1`  | `byte`                               | snr of the last frame              |
```
Comment (DeadByElpy) [inline docs/obis-observer/commands/GetLorawanState.md:35]:
```suggestion
| `1`  | `byte`                               | SNR of the last frame              |
```

File: docs/obis-observer/commands/GetLorawanState.md (hunk 1 above)
Comment (Pnom) [reply docs/obis-observer/commands/GetLorawanState.md]: fixed

File: docs/obis-observer/commands/GetLorawanState.md (hunk 2 above)
Comment (Pnom) [reply docs/obis-observer/commands/GetLorawanState.md]: fixed

File: docs/obis-observer/commands/GetDate.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+
+
+### Examples
+
+#### Device information:
+
+| Field             | Value                   | Hex          |
+| ----------------- | ----------------------- | ------------ |
+| command id        | `35`                    | `0x23`       |
```
Comment (DeadByElpy) [inline docs/obis-observer/commands/GetDate.md:44]:
```suggestion
| command id        | `36`                    | `0x24`       |
```

File: docs/obis-observer/commands/GetDate.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+| request id        | `7`                     | `0x07`       |
+| time              | `26.06.23 11:50:11 GMT` | `0x31cd2f73` |
+| uptime in seconds | `4016`                  | `0x00000fb0` |
+
+Message hex dump: `23 05 31 cd 2f 73 00 00 0f b0`
```
Comment (DeadByElpy) [inline docs/obis-observer/commands/GetDate.md:49]:
```suggestion
Message hex dump: `24 07 31 cd 2f 73 00 00 0f b0`
```

File: docs/obis-observer/commands/GetLorawanState.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+Request/response to get the Lorawan related state and statistic.
+
+
+## Request
+
+### Format
+
+| Size  | Type                                 | Field                              |
+| ----- | ------------------------------------ | ---------------------------------- |
+| `1`   | `byte`                               | command id = `0x24`                |
```
Comment (DeadByElpy) [inline docs/obis-observer/commands/GetLorawanState.md:12]: 0x24 already used at GetDateResponse

**Review by DeadByElpy — CHANGES_REQUESTED** [review]

File: docs/obis-observer/commands/GetDate.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+| Size  | Type                                 | Field                              |
+| ----- | ------------------------------------ | ---------------------------------- |
+| `1`   | `byte`                               | command id = `0x22`                |
+| `1`   | [Request ID](../types.md#request-id) | request/response unique identifier |
+
+### Examples
+
+| Field      | Value | Hex    |
+| ---------- | ----- | ------ |
+| command id | `35`  | `0x22` |
```
Comment (DeadByElpy) [inline docs/obis-observer/commands/GetDate.md:19]:
```suggestion
| command id | `34`  | `0x22` |
```

File: docs/obis-observer/commands/GetReadoutState.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+| `1`  | `byte`                               | the count of the parity errors                                                                          |
+| `1`  | `byte`                               | the count of the frame errors                                                                           |
+| `1`  | `byte`                               | the count of the overrun errors                                                                         |
+
+
+### Examples
+
+| Field                                                                                                   | Value | Hex          |
+| ------------------------------------------------------------------------------------------------------- | ----- | ------------ |
+| command id                                                                                              | `37`  | `0x25`       |
```
Comment (DeadByElpy) [inline docs/obis-observer/commands/GetReadoutState.md:49]:
```suggestion
| command id                                                                                              | `39`  | `0x27`       |
```

File: docs/obis-observer/commands/readme.md [hunk 8]
```diff
@@ -49,6 +52,9 @@ There are `2` types of command direction:
...
 | `0x1f` | [GetLorawanInfo](./GetLorawanInfo.md#response)                                      | Response to the [GetLorawanInfo](./GetLorawanInfo.md#request) downlink command.                                   |
 | `0x21` | [GetDeviceInfo](./GetDeviceInfo.md#response)                                        | Response to the [GetDeviceInfo](./GetDeviceInfo.md#request) downlink command.                                     |
+| `0x23` | [GetDate](./GetDate.md#response)                                                    | Response to the [GetDate](./GetDate.md#request) downlink command.                                     |
+| `0x24` | [GetLorawanState](./GetLorawanState.md#response)                                    | Response to the [GetLorawanState](./GetLorawanState.md#request) downlink command.                                     |
+| `0x26` | [GetReadoutState](./GetReadoutState.md#response)                                    | Response to the [GetReadoutState](./GetReadoutState.md#request) downlink command.                                     |
```
Comment (DeadByElpy) [inline docs/obis-observer/commands/readme.md:57]:
```suggestion
| `0x27` | [GetReadoutState](./GetReadoutState.md#response)                                    | Response to the [GetReadoutState](./GetReadoutState.md#request) downlink command.                                     |
```

**Review by DeadByElpy — CHANGES_REQUESTED** [review]

File: docs/obis-observer/commands/readme.md [hunk 9]
```diff
@@ -49,6 +52,9 @@ There are `2` types of command direction:
 | `0x1d` | [Reboot](./Reboot.md#response)                                                      | Response to the [Reboot](./Reboot.md#request) downlink command.                                                   |
 | `0x1f` | [GetLorawanInfo](./GetLorawanInfo.md#response)                                      | Response to the [GetLorawanInfo](./GetLorawanInfo.md#request) downlink command.                                   |
 | `0x21` | [GetDeviceInfo](./GetDeviceInfo.md#response)                                        | Response to the [GetDeviceInfo](./GetDeviceInfo.md#request) downlink command.                                     |
+| `0x23` | [GetDate](./GetDate.md#response)                                                    | Response to the [GetDate](./GetDate.md#request) downlink command.                                     |
+| `0x24` | [GetLorawanState](./GetLorawanState.md#response)                                    | Response to the [GetLorawanState](./GetLorawanState.md#request) downlink command.                                     |
```
Comment (DeadByElpy) [inline docs/obis-observer/commands/readme.md:56]:
```suggestion
| `0x25` | [GetLorawanState](./GetLorawanState.md#response)                                    | Response to the [GetLorawanState](./GetLorawanState.md#request) downlink command.                                     |
```

**Review by DeadByElpy — CHANGES_REQUESTED** [review]

File: docs/obis-observer/commands/GetDate.md (hunk 6 above)
Comment (Pnom) [reply docs/obis-observer/commands/GetDate.md]: fixed

**Review by DeadByElpy — APPROVED** [review]

**Pull request merged**